script/Arcueid/Day 8/08_00_ARC08_7_2.txt

```diff
@@ -145,7 +145,7 @@ What was that?
 -- Page 479, Offset 20923.
 -- 街灯と街灯の中間。明かりの届かない暗がりに、何かが立っているような―――
 There, in the space between two streetlights.
- In that pool of darkness the light can't quite reach, stands a figure―――
+ In that pool of darkness where the light can't quite reach, stands a figure―――
```

Comment:
either 'that' or 'where' is implicit here, but I don't see any harm in being explicit.